config/routes.rb

```diff
@@ -1,5 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'users#index'
```

Comment:
This change to `root` occured in the previous commit. Can you squash this removal into the previous commit?